README.md

```diff
@@ -1,6 +1,6 @@
 # rspec - a BDD test harness that works with stable Rust
 
-[![Build Status](https://travis-ci.org/mackwic/rspec.svg?branch=master)](https://travis-ci.org/mackwic/rspec) [![Coverage Status](https://coveralls.io/repos/github/mackwic/rspec/badge.svg)](https://coveralls.io/github/mackwic/rspec) [![Crates.io](https://img.shields.io/crates/v/rspec.svg?maxAge=2592000)](https://crates.io/crates/rspec) [![Crates.io](https://img.shields.io/crates/l/rspec.svg?maxAge=2592000)](https://github.com/mackwic/rspec/blob/master/LICENSE)
+[![Build Status](https://travis-ci.org/mackwic/rspec.svg?branch=master)](https://travis-ci.org/mackwic/rspec) [![Coverage Status](https://coveralls.io/repos/github/mackwic/rspec/badge.svg)](https://coveralls.io/github/mackwic/rspec) [![Crates.io](https://img.shields.io/crates/v/rspec.svg?maxAge=2592000)](https://crates.io/crates/rspec) [![Crates.io](https://img.shields.io/crates/l/rspec.svg?maxAge=2592000)](https://github.com/rust-rspec/rspec/blob/master/LICENSE)
```

Comment:
please change the other badges

Reply:
As I said:

> However, this PR doesn't touch anything about the badges, since I've no idea what is used and how Travis/Coveralls handle repositories renaming.

And after taking a look at it after your request:

- Travis: neither https://travis-ci.org/mackwic/rspec nor https://travis-ci.org/rust-rspec/rspec works, at least for me
- Coveralls: https://coveralls.io/github/mackwic/rspec works but https://coveralls.io/github/rust-rspec/rspec doesn't
- The Crates badge only refers the crates so it's not impacted

So I don't think there's anything I can change at this moment